recipes/cdst/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "cdst" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  noarch: python
```

Comment:
A package cannot be noarch *and* have compiled modules using cython.

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

However, I'm not sure that this package compiles anything. Why does it depend on cython?